include/envoy/upstream/cluster_manager.h

```diff
@@ -126,7 +126,7 @@ struct ClusterConnectivityState {
  * Cluster manager is initialized in two phases. In the first phase which begins at the construction
  * all primary clusters (i.e. with endpoint assignments provisioned statically in bootstrap,
  * discovered through DNS or file based CDS) are initialized. This phase may complete synchronously
- * with cluster manager construction iff all clusters are STATIC and without health checks
```

Comment:
im wondering whether `iff` was intentional - it has the meaning of "if and only if"

@yanavlasov ?

Reply:
ok,got it...